web/src/components/pair/PairCodeDiff.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="editor">
+    <v-row>
+      <v-col class="editor-title text--primary">
+        <v-icon left>mdi-file-outline</v-icon>
+        {{ props.pair.leftFile.shortPath }}
+      </v-col>
+
+      <v-col class="editor-title text--primary">
+        <v-icon left>mdi-file-outline</v-icon>
+        {{ props.pair.rightFile.shortPath }}
+      </v-col>
+    </v-row>
+
+    <div ref="editorElem" class="editor-element"></div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { withDefaults, ref, shallowRef, onMounted, watch, onUnmounted } from "vue";
+import { Pair, Metadata } from "@/api/models";
+import * as monaco from "monaco-editor";
+
+interface Props {
+  pair: Pair;
+  metadata: Metadata;
+}
+
+const props = withDefaults(defineProps<Props>(), {});
+
+// Editor template ref
+const editorElem = ref();
+// Monaco editor
+const editor = shallowRef();
+
+// Initialize the editor.
+const initialize = (): void => {
+  // Monaco file models.
+  const leftFileModel = monaco.editor.createModel(props.pair.leftFile.content, props.metadata.language);
+  const rightFileModel = monaco.editor.createModel(props.pair.rightFile.content, props.metadata.language);
+
+  // Monaco diff editor
+  editor.value = monaco.editor.createDiffEditor(editorElem.value, {
+    enableSplitViewResizing: false, // Do not allow resizing of the diff view.
+    readOnly: true,
+    automaticLayout: true,

```

Comment:
Monaco's custom menu does not add anything of value I think, so could you add `contextmenu: false` here?